docs/guides/how_to_guides/creating_batches/how_to_create_a_batch_request_using_an_active_data_connector.rst

```diff
@@ -0,0 +1,269 @@
+
+  .. code-block:: python
+
+    import great_expectations as ge
+    from great_expectations.cli.datasource import sanitize_yaml_and_save_datasource
+    from great_expectations.core.batch import BatchRequest
+
+    context = ge.get_context()
+
+
+2. **Configure a Datasource**
+
+  In the following configuration, a Datasource is configured with a ``PandasExecutionEngine`` and ``ConfiguredAssetFilesystemDataConnector``. The DataConnector is configured with a single DataAsset named ``my_reports``. It has the ``base_directory`` set to ``reports/`` and the regex ``pattern`` is set to capture two ``group_names``, ``name`` and ``num``.
+
+  .. code-block:: python
+
+    config = f"""
+      name: mydatasource
+      class_name: Datasource
+      execution_engine:
+        class_name: PandasExecutionEngine
+      data_connectors:
+        my_data_connector:
+          module_name: great_expectations.datasource.data_connector
+          class_name: ConfiguredAssetFilesystemDataConnector
+          base_directory: /
+          glob_directive: "*.csv"
+          assets:
+            my_reports:
+              base_directory: reports/
+              pattern: (.+)-(\\d.*)\\.csv
+              group_names:
+                - name
+                - num
+
+    """
+
+
+3. **(Optional) run** ``test_yaml_config()`` **to ensure that your configuration is working.**
+
+  .. code-block:: python
+
+    context.test_yaml_config(
+        yaml_config=config
+    )
+
+
+  If the configuration is correct you should see output similar to this.  Also, your DataConnector will now be active.
+
+  .. code-block:: bash
+
+    Attempting to instantiate class from config...
+      Instantiating as a Datasource, since class_name is Datasource
+      Successfully instantiated Datasource
+
+    ExecutionEngine class name: PandasExecutionEngine
+    Data Connectors:
+      my_data_connector : ConfiguredAssetFilesystemDataConnector
+
+      Available data_asset_names (1 of 1):
+        my_reports (3 of 5): ['a-1.csv', 'b-2.csv', 'c-3.csv']
+
+        Unmatched data_references (0 of 0): []
+
+
+4. **Save Configuration**
+
+  .. code-block:: python
+
+    sanitize_yaml_and_save_datasource(context, config, overwrite_existing=False)
+    context = ge.get_context()
+
+5. **Obtain an ExpectationSuite**
+
+  Your DataContext can be used to create or retrieve an ExpectationSuite.
+
+  .. code-block:: python
+
+    suite = context.get_expectation_suite("insert_your_expectation_suite_name_here")
+
+  Alternatively, if you have not already created a suite, you can do so now.
+
+  .. code-block:: python
+
+    suite = context.create_expectation_suite("insert_your_expectation_suite_name_here")
+
+
+6. **Construct a** ``BatchRequest``.
+
+  The following ``BatchRequest`` will retrieve a Batch corresponding to ``a-1.csv`` by using ``batch_filter_parameters`` as a ``data_connector_query``.  Additional examples of ``data_connector_query`` like ``limit` and ``index`` can be found below.
```

Comment:
Maybe just a note here or below to make explicit that we want to make sure we are selecting exactly one batch (so for instance, we may want to specify both "name" and "num").

(I'm saying this because I tested this on the taxi data where the "name" for both files was the same, so I had to do exactly that :) )